src/public.js

```diff
@@ -39,7 +39,9 @@ const hasToken = (document.getElementById('sharingToken') !== null);
 const isPasswordProtected = (document.getElementById('password-submit') !== null);
 
 if (hasToken && !isPasswordProtected) {
-	$('body').prepend($('<div>').attr('id', 'terms_of_service_confirm'));
+	const tofc = document.createElement('div');
```

Comment:
tofc? :stuck_out_tongue_closed_eyes: 

But nice PR in general, is that all jquery usage?

Reply:
Mind to fix it to tosc?